k8s/aat/common/bsp/bulk-scan-sample-app.yaml

```diff
@@ -21,6 +21,31 @@ spec:
       replicas: 2
       useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/bulk-scan/sample-app:prod-49b7c377
+      testsConfig:
+        memoryLimits: "3072Mi"
+        serviceAccountName: tests-service-account
+        keyVaults:
+          bulk-scan:
+            resourceGroup: bulk-scan
+            secrets:
+              sample-app-s2s-secret: S2S_SECRET
+        environment:
+          TEST_URL: "http://bulk-scan-sample-app-aat.service.core-compute-aat.internal"
+          SLACK_CHANNEL: "bsp-test-notices"
+          SLACK_NOTIFY_SUCCESS: true
+          TEST_S2S_NAME: "bulk_scan_sample_app_tests"
+          TEST_S2S_URL: "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
+          TEST_SCAN_DELAY: "4000"
+        functionaltestscron:
+          enabled: true
+          image: hmctspublic.azurecr.io/bulk-scan/sample-app-test:prod-49b7c377
+          schedule: "15/30 6-19 * * mon-fri"
```

Comment:
This bit should go

Reply:
I misunderstood.

Reply:
this block should go, not cron definition ;)